src/components/CommonPageItems/index.js

```diff
@@ -1,32 +1,34 @@
 import React from 'react';
 import './index.css';
-import { Link } from 'react-router-dom';
```

Comment:
In case if "react-router-dom" is not required anymore, could you please remove it from package.json dependecies?